### PR TITLE
Add key points

### LINKED ISSUE #48: update keypoints
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this issue is about a specific episode within a lesson, please provide its link or filename.

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to respond to your contribution.
To ensure that the right people get notified, you can tag some of the last contributors with `@githubname`.

Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact the HSF training convenors ([contacts here](https://hepsoftwarefoundation.org/workinggroups/training.html)).

You may delete these instructions from your issue.

\- HSF Training
</details>

### PR DESCRIPTION
**Pull request by amanmdesai**
Closes #48 

Episodes
- [x] 1
- [x] 2
- [x] 3
- [x] 4
- [x] 5
- [x] 6

Not much required after episode 6

### CODE REVIEW

**Comment by amanmdesai** [comment]
No changes for 5, 
7 is coffee break
Not adding anything  after episode 6
Doubt is whether to use image id or container id -- but otherwise i have tried to give the basic command

File: _episodes/03-running-containers.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ objectives:
 - "Understand container state"
 - "Stop and restart containers"
 keypoints:
-- "Run containers with `docker run`"
+- "Run containers with `docker run <image-id>`"
 - "Monitor containers with `docker ps`"
-- "Exit interactive sessions just as you would a shell"
+- "Exit interactive sessions just using the `exit` command"
```
Comment (klieret) [inline _episodes/03-running-containers.md]: I'd remove the "just" here

File: _episodes/04-file-io.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ objectives:
 - "Copy files to and from the docker container"
 - "Mount directories to be accessed and manipulated by the docker container"
 keypoints:
-- "We can copy files with ``docker cp``"
-- "We can mount volumes with ``docker run -v <path on host>:<path in container> <image>``"
+- "Copy files with ``docker cp``"
```
Comment (klieret) [inline _episodes/04-file-io.md]: These should be single backticks though it'll probably render OK

**Review by klieret — CHANGES_REQUESTED** [review]
Looks good except for the small details :)

File: _episodes/03-running-containers.md (hunk 1 above)
Comment (amanmdesai) [reply _episodes/03-running-containers.md]: done

File: _episodes/04-file-io.md (hunk 2 above)
Comment (amanmdesai) [reply _episodes/04-file-io.md]: Done

**Review by klieret — APPROVED** [review]

**Pull request merged**